nuocd/nuodb_adminquery.py

```diff
@@ -170,7 +170,6 @@ def get_admin_conn(pid):
                 del running_local_processes[key]
 
     except subprocess.CalledProcessError:
-        print_('nuodb not running', file=sys.stderr)
```

Comment:
You might want to add print or log when a new nuodb process is found and when it exits (might already be there).  that way lack of message is your trigger that something went wrong with sharedProcessSpace.